app/controllers/concerns/curation_concerns/catalog_controller.rb

```diff
@@ -14,7 +14,6 @@ module CurationConcerns::CatalogController
     # This applies appropriate access controls to all solr queries
     CatalogController.search_params_logic += [:add_access_controls_to_solr_params]
     self.search_params_logic += [:filter_models]
-    self.search_params_logic += [:only_curation_concerns]
```

Comment:
@escowles what is doing the duplicate filtering?  Seems strange to remove a line that was adderd by @jcoyne just a few days ago.